templates/zerver/register.html

```diff
@@ -74,14 +79,17 @@ <h1>You&rsquo;re almost there.</h1>
                             {% endfor %}
                         {% endif %}
                     </div>
-                    <div class="help-box margin-top">
-                        {{ _("The address you'll use to log in to your organization.") }}
+                    <div class="help-text">
+                        {{ _("The URL users will use to access the new organization.") }}
                     </div>
                 </div>
                 {% endif %}
             </section>
 
             <section class="user-registration">
+                {% if creating_new_team %}
+                <legend>{{ _('Your account') }}</legend>
```

Comment:
According to the HTML standard a `<legend>` tag must be the first child of a `<fieldset>` tag. Changing these sections to fieldsets would probably also be more semantically accurate.

Reply:
Fixed this.